### PR TITLE
HLA-1111: Updated SVM filenames for WFPC2

### LINKED ISSUE #1662: Update WFPC2 HAP filenames
_Issue [HLA-1111](https://jira.stsci.edu/browse/HLA-1111) was created on JIRA by [Jennifer Mack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mack):_

As discussed in team_burgundy, we would like to tweak the WFPC2 file names slightly so that the second instance of 'wfpc2' is replaced with 'pc'.

     hst_6192_b1_wfpc2_{*}wfpc2{*}_f814w_u2xgb1_drz.fits   <-- old name
     hst_6192_b1_wfpc2_{*}pc{*}_f814w_u2xgb1_drz.fits         <-- new name

### PR DESCRIPTION
**Pull request by s-goldman**
Resolves [HLA-1111](https://jira.stsci.edu/browse/HLA-1111)

Closes #1662 

This PR changes SVM filenames of WFPC2 products. The current SVM filenames follow the following structure:

`hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_dr[cz].fits`

where previously the `<instr>` and `<detector>` were both WFPC2. With these changes, the `<detector>` is forced to be "pc" if the `<instr> `is "WFPC2".

### CODE REVIEW

**Comment by s-goldman** [comment]
I also had to change mentions of wfpc2_wfpc2 in filenames and json files to wfpc2_pc for the code to work properly.

**Comment by s-goldman** [comment]
> @s-goldman Can you please check PR#1473 where Warren changed the names to include ...wfpc2_wfpc2... In his case he was eliminating both the use of _wf_ and _pc_. While I can see there are some unrelated changes his PR, it is good to check this PR to make sure all the necessary changes have been addressed by this PR. For example, the environment variable needed for SVM catalog generation for WFPC2 has not been updated in hapsequencer.py in this PR.

I've reverted the changes related to WFPC2 from #PR1473 and the code passes the jenkins regression tests. The github actions tests keep seeming to fail (on both of my current PRs), but the failures seem related to connection issues:

`Error: Connection broken: IncompleteRead(1061384 bytes read, 9591736 more expected)'`

It doesn't seem related to these code changes, but I could be wrong.

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
Sorry (!) I know this is tedious, but I think we need to keep the nomenclature and file structure consistent across all instruments.  It looks like the [svm | mvm]_parameters need to have the subdirectories for WFPC2 renamed to wfpc2/pc/*.json.  This will also require changing the contents of the */index files.

**Comment by s-goldman** [comment]
> Sorry (!) I know this is tedious, but I think we need to keep the nomenclature and file structure consistent across all instruments. It looks like the [svm | mvm]_parameters need to have the subdirectories for WFPC2 renamed to wfpc2/pc/*.json. This will also require changing the contents of the */index files.

No worries, I totally agree. Thanks for catching that. I've made the change in the most recent commit.

File: drizzlepac/pars/hap_pars/mvm_parameters/wfpc2_pc_index.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (mdlpstsci) [inline drizzlepac/pars/hap_pars/mvm_parameters/wfpc2_pc_index.json:1]: The filename paths in this file have to be updated.

File: drizzlepac/pars/hap_pars/svm_parameters/wfpc2_pc_index.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (mdlpstsci) [inline drizzlepac/pars/hap_pars/svm_parameters/wfpc2_pc_index.json:1]: The filename paths in this file have to be updated.

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
The index file contents have to be updated, and then it is ready to go.

**Review by mdlpstsci — APPROVED** [review]
Lots of nooks and crannies for this ticket!

**Pull request merged**